### PR TITLE
chip

### LINKED ISSUE #33: Add Chip component

### PR DESCRIPTION
**Pull request by edsonsnunes**
fix #33

### CODE REVIEW

**Comment by iurynogueira** [comment]
Faltou o disabled

File: projects/ion/src/lib/chip/chip.component.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<button (click)="select()" [ngClass]="{
+    chip: !selected,
+    'chip-selected': selected,
+    'chip-md': size === 'md',
+    'chip-disabled' : disabled}">
```
Comment (iurynogueira) [inline projects/ion/src/lib/chip/chip.component.html:5]:
```suggestion
>
```

Comment (iurynogueira) [reply projects/ion/src/lib/chip/chip.component.html:5]: Add [disabled]="disabled" in HTML tag and inner CSS add button[disabled] { style.... }

**Review by iurynogueira — CHANGES_REQUESTED** [review]

**Review by iurynogueira — APPROVED** [review]
nice work!

**Review by Deeved — APPROVED** [review]

**Pull request merged**